server/server.js

```diff
@@ -10,6 +11,8 @@ app.use(express.urlencoded({ extended: true }));
 app.use(express.static(path.resolve(__dirname + '../src')));
 app.use('/build', express.static(path.join(__dirname + '../build')));
 
+app.use('/error', errorRouter);
```

Comment:
Change `error` to `errors` and `errorRouter` to `errorsRouter`: https://blog.dreamfactory.com/best-practices-for-naming-rest-api-endpoints/#:~:text=Use%20nouns%20for%20naming%20URIs,%3A%2F%2Fapi.example.com.&text=In%20general%2C%20using%20plural%20nouns,admin%20for%20the%20administrative%20user).